man/systemd.netdev.xml

```diff
@@ -527,9 +527,9 @@
 
     <variablelist class='network-directives'>
       <varlistentry>
-        <term><varname>Id=</varname></term>
+        <term><varname>VNI=</varname></term>
         <listitem>
-          <para>The VXLAN ID to use.</para>
+          <para>The VXLAN Network Identifier (or VXLAN Segment ID). Ranges [0-16777215].</para>
```

Comment:
A number in the range 0..16777215.

Reply:
HA number ranges [1-16777215] ?

Reply:
I prefer `Takes a number in the range [1-16777215].` or `Takes a number between 1 and 16777215.`